### PR TITLE
LPS-152582 [Raylife AP] Create Commission Chart Layout in Dashboard Page

### PR DESCRIPTION
**Pull request by joanagrego**

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+.bb-chart-arcs-title {
+	dominant-baseline: middle;
+	font-size: 1.3em;
+}
+
+.bb-chart-arcs .bb-chart-arcs-background {
+	fill: #e0e0e0;
+	stroke: none;
+}
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css:31]: Add a line break after `}`

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+.line-chart-container .line-chart-legend {
+	flex-direction: column;
+}
+
+@media screen and (max-width: 1280px) {
+	.line-chart-legend {
+		flex-direction: row !important;
+		justify-content: space-evenly !important;
+	}
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css:54]: Add a line break after `}`

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+	fill: #000;
+}
+
+.bb path,
+.bb line {
+	fill: none;
+}
+
+.line-chart-container .line-chart-legend {
+	flex-direction: column;
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css:47]: Please, use bootstrap for this.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+	dominant-baseline: middle;
+	font-size: 1.3em;
+}
+
+.bb-chart-arcs .bb-chart-arcs-background {
+	fill: #e0e0e0;
+	stroke: none;
+}
+.bb-chart-arcs .bb-chart-arcs-gauge-unit {
+	fill: #000;
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css:33]: Please, use a variable for `#000`

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+	font-size: 24px;
+}
+
+.bb-chart-arc .bb-gauge-value {
+	fill: #000;
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css:38]: Please, use a variable for `#000`

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+	font-size: 32px;
+	font-weight: 700;
+}
+
+.line-chart-container .line-chart-tooltip {
+	border: 1px solid #55c2ff;
+	color: #55c2ff;
+	font-size: 0.875rem;
+	font-style: normal;
+	width: 100%;
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css:20]: Could you use `w-100` bootstrap property for this?

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.css [hunk 7]
```diff
@@ -0,0 +1,36 @@
+.dashboard-commission-container {
+	min-height: 257px;
+}
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.css:3]: Add a line break after this `}`

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.css [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+.dashboard-commission-container
+	.dashboard-commission-header
+	.dashboard-commission-title::after {
+	background-color: #4d85ff;
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.css:7]: Please, use a variable for this.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.css [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+	height: 1px;
+	left: 28px;
+	min-width: 20px;
+	position: absolute;
+	top: 50px;
+}
+
+.dashboard-commission-container .dashboard-commission-select {
+	width: 105px;
+	border-color: #fff;
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.css:17]: Please, use a variable for this.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.css [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+	background-color: #4d85ff;
+	content: '';
+	height: 1px;
+	left: 28px;
+	min-width: 20px;
+	position: absolute;
+	top: 50px;
+}
+
+.dashboard-commission-container .dashboard-commission-select {
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.css:16]: Please, sort properties in alphabetical order.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.js [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+ * details.
+ */
+
+import {ClaySelect} from '@clayui/form';
+import React, {useState} from 'react';
+
+import LineChart from '../../../common/components/line-chart';
+
+export default function () {
+	const months = [
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.js:21]: This array should be a constant. Is it make sense for you?

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.js [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+		'June',
+		'July',
+		'August',
+		'September',
+		'October',
+		'November',
+		'December',
+	];
+
+	const d = new Date();
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.js:36]: Please, change the variable name.

**Review by silasbsi — CHANGES_REQUESTED** [review]
@joanagrego I added some points.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css (hunk 1 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css:30]: done!

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css (hunk 2 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css:50]: done

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css (hunk 5 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/common/components/line-chart/index.css]: done

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.css (hunk 9 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.css]: done

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.js (hunk 11 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-commission/index.js]: done

**Comment by joanagrego** [comment]
ci:test:sf

**Review by silasbsi — APPROVED** [review]

**Comment by silasbsi** [comment]
ci:forward

**Comment by joanagrego** [comment]
ci:forward

**Pull request closed without merging**